core/src/main/kotlin/com/strumenta/kolasu/language/KolasuLanguage.kt

```diff
@@ -87,7 +88,7 @@ class KolasuLanguage(val qualifiedName: String) {
             Any::class -> Unit
             else -> {
                 if (kClass.java.isInterface) {
-                    if (kClass.isMarkedAsNodeType()) {
+                    if (kClass.isMarkedAsNodeType() && !kClass.isSubclassOf(CommonElement::class)) {
```

Comment:
Maybe we can add a comment to specify that those subclasses are manually added in the StarLasu LW Language